codebuild/bin/install_ubuntu_dependencies.sh

```diff
@@ -39,9 +39,10 @@ base_packages() {
   apt install -y software-properties-common
   add-apt-repository ppa:ubuntu-toolchain-r/test -y
   add-apt-repository ppa:longsleep/golang-backports -y
+  add-apt-repository ppa:deadsnakes/ppa -y # needed for python3.9 and python3.9-distutils
   apt-get update -o Acquire::CompressionTypes::Order::=gz
```

Comment:
I thought we discussed changing the default python being super discouraged?

Reply:
I don't believe this is changing the default python, its just installing another python version that exists independently

Reply:
```bash
$ python3 --version
Python 3.6.9
$ python3.9 --version
Python 3.9.13
```